### PR TITLE
Force global

### LINKED ISSUE #27: Provide some way of making global pairing the default
It's useful to have the choice between setting a pair for a particular repo, and setting it globally. But in my experience, you have to consistently do one or the other - they don't mix well. I keep getting into situations where i'm committing as someone who worked on the machine the previous day, but set their pair differently. I could fix this by having a more effective brain, but realistically, that's not going to happen.

So, what i'd like to do is have some way of forcing global pairing on a machine. So if someone writes `git pair kb wc`, it actually sets a global pair. That way, there's only global pairing, and no confusion.

Forcing global pairing, rather than local pairing, makes more sense for us, as we often have multi-repo projects, with a pair at one machine committing to multiple repositories in the same session.

If you think this is a sane feature, and if i can find some way of implementing it sanely, i am happy to submit a PR.

The best thing i've thought of so far is an environment variable. It could be GIT_DUET_GLOBAL, which forces globalness if set. Or it could be GIT_DUET_OPTS, which is added to the command line, so it could contain other options (if there are ever any). How does that sound?

### PR DESCRIPTION
**Pull request by tomwhoiscontrary**
This is my attempt to resolve #27.

The code i added to Cli took it over Rubocop's class size limit, so i split the parsing into a separate class. I'm happy to go back and do this a different way if you prefer.

Let me know what you think!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +301,11 @@ github](https://github.com/meatballhat/git-duet/issues/new).
 
 1. Fork it
 2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git dci -am 'Added some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+3. Implement!
+4. Run the tests
```
Comment (jszwedko) [inline README.md]: :+1:

File: lib/git/duet/cli_options.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+module Git
+  module Duet
+    class CliOptions
```
Comment (jszwedko) [inline lib/git/duet/cli_options.rb]: :+1:

**Comment by jszwedko** [comment]
This seems good to me!

**Comment by meatballhat** [comment]
Yay!  Thanks, @tomwhoiscontrary!

**Pull request merged**